addon/components/frost-select.js

```diff
@@ -257,6 +257,19 @@ export default Component.extend({
     if (width) styles += `width: ${width}px; max-width: initial; min-width:initial; `
     return Ember.String.htmlSafe(styles)
   },
+
+  @readOnly
+  @computed('items', 'selectedItems')
+  /**
+   * As of Ember 2.10, we have to make this a computed property to tell Ember to update our frost-select-dropdown.
+   * Any properties that might be updated after the initial render should be observed by this property.
+   * (@theotherdude 12/20/2017)
+   * @returns {string} 'frost-select-dropdown'
+   */
+  frostSelectDropdownName () {
```

Comment:
OK, I get it... but does it have to be this property? Couldn't we make new computed props that pass along items and selectedItems? I think that would be more sensible than returning a hard-coded string from an otherwise unrelated computed prop just to get them observed.

Reply:
John and I talked about this in person. It doesn't seem to matter how we pass the named arguments to the component helper, we only trigger a fresh render when we update the positional argument. 

The tests I added fail without this change.